util/src/main/java/io/kubernetes/client/util/Yaml.java

```diff
@@ -357,8 +360,11 @@ protected NodeTuple representJavaBeanProperty(
   }
 
   /** @return An instantiated SnakeYaml Object. */
-  public static org.yaml.snakeyaml.Yaml getSnakeYaml() {
```

Comment:
this is a public method, should we mark it as deprecated and keep this for 1~2 releases for backward-compatibility?

Reply:
Done. (though this should really be private)